src/Components/benchmarkapps/Wasm.Performance/TestApp/wwwroot/benchmarks/blazorDownloadSize.js

```diff
@@ -0,0 +1,15 @@
+export function getBlazorDownloadSize() {
```

Comment:
Again, if there's a better way to do this, I'd be happy to use it. This launches a page in an iframe that wires up the new hook that I added. The contents of the iframe are slightly different because we need to hook up the callback after Blazor's scripts have been initialized, but before downloads happen. The instance of chrome in docker does caching correctly, so relying on caching to happen isn't problematic.